src/reference-designs/tkg-edge.md

```diff
@@ -0,0 +1,885 @@
+- Network Policy: Network policies allow you to use preconfigured templates to define how pods communicate with each other and other network endpoints.
+- Quota Policy: Quota policies allow you to constrain the resources used in your clusters as aggregate quantities across specified namespaces, using pre-configured and custom templates. For more information, see [Managing Resource Consumption](https://docs.vmware.com/en/VMware-Tanzu-Mission-Control/services/tanzumc-using/GUID-1905352C-856F-4D06-BB86-426F90486C32.html).
+- Security Policy: Security policies allow you to manage the security context in which deployed pods operate in your clusters by imposing constraints on your clusters that define what pods can do and which resources they have access to. For more information, see [Pod Security Management](https://docs.vmware.com/en/VMware-Tanzu-Mission-Control/services/tanzumc-concepts/GUID-6C65B33B-C1EA-465D-B909-3C4F51704C1A.html).
+- Custom Policy: Custom policies allow you to implement additional business rules, using templates that you define, to enforce policies that are not already addressed using the other built-in policy types. For more information, see [Creating Custom Policies](https://docs.vmware.com/en/VMware-Tanzu-Mission-Control/services/tanzumc-using/GUID-1FF7A1E5-8456-4EF4-A532-9CF31BE88EAA.html).
+
+Some policies mentioned here are not available in Tanzu Mission Control Standard edition. For more information, see the [VMware Tanzu Mission Control Feature Comparison Chart](https://tanzu.vmware.com/content/tanzu-mission-control/tmc-comparison-chart).
+
+#### Policy Inheritance
+
+In the Tanzu Mission Control resource hierarchy, you can specify policies at the following three levels:
+
+- Organization
+- Object groups (cluster groups and workspaces)
+- Kubernetes objects (clusters and namespaces)
+
+#### Global Visibility example
+
+With TMC you can group clusters based on different types of locations and manage fleets of clusters instead of managing them individually. When we are dealing with large scale edge deployments this becomes very critical as customers might have two or three different types of locations. Being able to group them with cluster groups, looking at individual fleet health, performing collective operations is key for edge deployments. Figure X shows an overview of the cluster group.
+
+![](./img/tkg-edge/ce29c1e20bf0ecc1ffe2dfdc658cb585.png)
+
+Figure: Managing the edge cluster fleet using TMC
+
+#### Policy Management example
+
+Policies allow you to provide a set of rules that govern your organization and all the objects it contains. Policies can be applied to fleets of clusters by defining once to ensure that the kubernetes clusters and applications are secure, compliant while enabling self service which makes it very appealing for edge scale.  Tanzu mission control provides following policy types that can be applied to fleets of clusters:
```

Comment:
Policies allow ... ensure that the Kubernetes clusters and ... Tanzu Mission Control provides ...